src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -775,6 +775,34 @@ class npc_salanar_the_horseman : public CreatureScript
     }
 };
 
+enum HorseSeats
+{
+    SEAT_ID_0                       = 0
```

Comment:
Out of curiosity, wouldn't it be enough to name the enum `SEAT_ID` when there is only 1 seat? Or have I missed something?
BTW: with no nearby enums with similar spacing, I suggest keeping the space between enum name and `=` between 1 and 4.

Reply:
Since we are accessing seat through specific ID I think this is little bit more readable option.